Smarty/templates/modules/Documents/pdfViewer.html

```diff
@@ -70,6 +94,7 @@
       </div>  <!-- sidebarContainer -->
 
       <div id="mainContainer">
+        <i id="dragSignature" class="fa-solid fa-signature fa-2x siginaturePositioning hidden" title="Signature"></i>
```

Comment:
this is a bootstrap icon I suppose and the reason we need bootstrap?  can we do that with LDS? are there others?

Reply:
@joebordes - It's a font awesome.